app/components/api-fetch-container/playlists-grid.tsx

```
@@ -1,25 +1,96 @@
-        playlist => (
-          <PlaylistCard
-            key={playlist.id}
-            id={playlist.id}
-            title={playlist.title}
-            description={playlist.description}
-            thumbnailurl={playlist.thumbnailurl}
+    <div className="grid grid-cols-2 md:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5 2xl:grid-cols-6 gap-4">
+      <button
+        onClick={() => setEditingPlaylist({ id: '', title: '', type: "Playlist", owner: {id: "", username: ""}, thumbnailurl: '' })}
+        className="aspect-square flex flex-col items-center justify-center rounded-lg bg-[--md-sys-color-surface-container] hover:bg-[--md-sys-color-surface-container-high] transition-colors gap-2"
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)